core/server/lib/members/index.js

```diff
@@ -74,11 +74,6 @@ module.exports = function MembersApi({
 
         const memberPromise = getMember({email});
 
-        memberPromise.catch(() => {
-            res.writeHead(200);
-            res.end();
-        });
-
```

Comment:
Removed this as it seemed redundant and responsible for error about re-triggering response after its sent once, we are handling `memberPromise` already below along with its error handling.

Reply:
I think you're right about this being part of a situation where headers get sent twice - though I don't think we want to remove it as it changes functionality which keeps emails private - the catch sends a 200 if the member doesn't exist rather than an error, I think we might wanna jiggle the promise chain to use error callbacks rather than catch